Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -125,11 +119,11 @@ public class BTShopperInsightsClient {
             paymentMethodsDisplayed: paymentMethodsDisplayedString
         )
     }
-    
-    /// Call this method when the Venmo button has been selected/tapped by the buyer.
-    /// This method sends analytics to help improve the Shopper Insights feature experience
-    public func sendVenmoSelectedEvent() {
-        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.venmoSelected)
+
+    /// Call this method when a button has been selected/tapped by the buyer.
+    /// This method sends analytics to help improve the Shopper Insights feature experience.
+    public func sendSelectedEvent(for buttonType: BTButtonType) {
+        apiClient.sendAnalyticsEvent(buttonType.rawValue)
```

Comment:
Depending on the order of merging in PRs we may needed to add the session ID work in PR #1485 here. Will want to coordinate with @scannillo on who merges in first and adds that.

Reply:
I realized once reviewing the ticket that it will conflict with the `presentment` PR, so I figured there will be discussion later about what to merge first